src/components/admin/MainIssueController.jsx

```diff
@@ -229,7 +434,7 @@ export default class MainIssueController extends FalcorController {
             primary
             style={styles.publishingButtons}
             onTouchTap={this.publishIssue}
-            disabled={published}
+            disabled={published || this.state.changed || this.state.saving}
```

Comment:
Let's also do it if we're publishing, that should already have been here, not your bad but lets fix it now